### ISSUE TITLE
Characters missing

### ISSUE THREAD

**Issue by dbass81** [VoxelBoy/MobileUO#9]
Connected to UO Renaissance after downloading client files and the character models are missing.

**Comment by codemonkey1** [comment]
特我3人与人图3是%


---原始邮件---
发件人: &quot;dbass81&quot;<notifications@github.com&gt;
发送时间: 2020年11月26日(周四) 上午7:47
收件人: &quot;VoxelBoy/MobileUO&quot;<MobileUO@noreply.github.com&gt;;
抄送: &quot;Subscribed&quot;<subscribed@noreply.github.com&gt;;
主题: [VoxelBoy/MobileUO] Characters missing (#9)




 
Connected to UO Renaissance after downloading client files and the character models are missing.
 
&mdash;

**Comment by dbass81** [comment]
> 特我3人与人图3是%
> […](#)
> ---原始邮件--- 发件人: &quot;dbass81&quot;<notifications@github.com&gt; 发送时间: 2020年11月26日(周四) 上午7:47 收件人: &quot;VoxelBoy/MobileUO&quot;<MobileUO@noreply.github.com&gt;; 抄送: &quot;Subscribed&quot;<subscribed@noreply.github.com&gt;; 主题: [VoxelBoy/MobileUO] Characters missing (#9) Connected to UO Renaissance after downloading client files and the character models are missing. &mdash; You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Sorry I don't understand.  I'm using the MobileUO app on an Android 9 phone, and everything works except for the character models in game are invisible.

**Comment by Jhobean** [comment]
![image](https://user-images.githubusercontent.com/51728381/100389892-b0d5d580-2ffc-11eb-9609-83a45c9eda00.png)

**Comment by computerkrieger** [comment]
i have the same issue . I can not see my char.

**Comment by iorimaluko** [comment]
I'm installing the game ..... I can log in more the characters are invisible

**Comment by popoverde** [comment]
yeah i had same issue, after 3 months digging it, found the problem causing it, and finally got fixed, however im not going to share shit, since the community is full of leechers and retards that never contribute. fuck yall. i aint even mad

**Comment by matty2023** [comment]
Did this ever get sorted as having same issue on kids phones and tablets but ok on my phone ??
